README.md

```diff
@@ -87,17 +87,17 @@ All released versions can be found [here](https://github.com/sonatype-nexus-comm
 | Hosted  |                      |
 | Group   |                      |
 
-NOTE: This is an early version of P2 Proxy and does not fully support the following as of yet:
+NOTE: P2 Proxy does not fully support the following as of yet:
 
-* [Composite Repositories](https://help.eclipse.org/neon/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Fguide%2Fp2_composite_repositories.htm)
-  * This is implemented but likely will fail if the composite files reference absolute paths rather than relative paths
 * Old style (non-P2) update sites. The code that accomplished this can be found [here](https://github.com/sonatype/nexus-public/blob/nexus-2.x/plugins/p2/nexus-p2-repository-plugin/src/main/java/org/sonatype/nexus/plugins/p2/repository/proxy/P2ProxyRepositoryImpl.java)
   * These are fairly old, and have not been added at all
 
 If you'd like it to support the aforementioned sites, please file an issue, or better yet, submit a PR :)
 
 ## Installing the plugin
 
+#### !!! After NXRM version 3.21, last released plugin's version is available directly from manager and no need to install it separately !!!
```

Comment:
NOTE: May be tbd but the phrasing "After NXRM version 3.21" does not include 3.21.  If p2 is to be included in 3.21, then it should be "In NXRM version 3.21 and beyond".

Suggestion: Make "from manager" into "in product".

Trivial: Make "and no need" to "and there is no need".

Reply:
Done e402d7282c7d5e04f122b32b85ea8422c3cd9fd8